cli/graphql.md

```diff
@@ -412,6 +412,147 @@ type Subscription {
 }
 ```
 
+### @key
+
+The `@key` directive makes it simple to configure custom index structures for `@model` types. 
+
+DynamoDB is a distributed hash table that can execute efficient range queries on extremely large data sets but doing so effectively takes a bit of forethought. DynamoDB query operations may use at most two attributes to efficiently query data. The first query argument passed to a query (the partition key) must use strict equality and the second attribute (the sort key) may use gt, ge, lt, le, eq, beginsWith, and between. Despite these restrictions, DynamoDB can effectively implement a wide variety of access patterns that are powerful enough for the majority of applications.
```

Comment:
Please reword this first sentence to use language appropriate with the Dynamo messaging: https://aws.amazon.com/dynamodb/

Reply:
I don't believe "partition" key is used any longer by Dynamo they use "primary"

Reply:
Remove "Despite these restrictions"

Reply:
Done